modules/services/dunst.nix

```diff
@@ -10,29 +16,54 @@ with lib;
       enable = mkEnableOption "the dunst notification daemon";
 
       settings = mkOption {
-        type = types.attrs;
+        type = types.attrsOf types.attrs;
         default = {};
         description = "Configuration written to ~/.config/dunstrc";
+        example = literalExample ''
+          {
+            global = {
+              geometry = "300x5-30+50";
+              transparency = 10;
+              frame_color = "#eceff1";
+              font = "Droid Sans 9";
+              browser = "''${config.firefox.package}/bin/firefox -new-tab";
```

Comment:
I think you'll need `browser = "''${config.programs.firefox.package}/bin/firefox -new-tab";` here. If dunst picks up commands from `PATH` then it should be enough to do `browser = "firefox -new-tab"`.

I've never actually gotten shortcuts to work in dunst so I don't know how the `browser` and `dmenu` options work :-(

Reply:
Same here :) Just added it as an example of usage.

Reply:
I've just deleted it not to confuse users :)